automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoTestUtils.scala

```diff
@@ -76,9 +76,13 @@ trait LeonardoTestUtils
   val getAfterCreatePatience = PatienceConfig(timeout = scaled(Span(5, Minutes)), interval = scaled(Span(2, Seconds)))
 
   val multiExtensionClusterRequest = UserJupyterExtensionConfig(
+    //TODO: Re-enable the below two lines when nbconvert is updated to 6.0.1
```

Comment:
can we use some other random extensions instead of disabling testing?

Reply:
Yeah these extensions were picked randomly, we could try some other ones if these are problematic. Ideas:

* [toc](https://jupyter-contrib-nbextensions.readthedocs.io/en/latest/nbextensions/toc2/README.html)
* [codefolding](https://jupyter-contrib-nbextensions.readthedocs.io/en/latest/nbextensions/codefolding/readme.html)
* [igv](https://github.com/igvteam/igv-jupyter#:~:text=igv%2Djupyter%20is%20an%20extension,its%20API%20from%20the%20notebook.)